build.sbt

```diff
@@ -3,22 +3,30 @@ import sbtassembly.AssemblyPlugin.autoImport.assembly
 import sbtassembly.{MergeStrategy, PathList}
 
 val scala212V = "2.12.10"
+val scala213V = "2.13.12"
+
+lazy val supportedScalaVersions = List(scala212V, scala213V)
+
+lazy val defaultScalaV = sys.env.get("NUSSKNACKER_SCALA_VERSION") match {
+  case None | Some("2.12") => scala212V
+  case Some("2.13")        => scala213V
+  case Some(unsupported)   => throw new IllegalArgumentException(s"Nu doesn't support $unsupported Scala version")
+}
 
 val scalaCollectionsCompatV = "2.9.0"
 
 // Silencer must be compatible with exact scala version - see compatibility matrix: https://search.maven.org/search?q=silencer-plugin
 // Silencer 1.7.x require Scala 2.12.11 (see warning above)
 val silencerV_2_12 = "1.6.0"
-val silencerV      = "1.7.0"
+val silencerV      = "1.7.17"
 
-val flink114V            = "1.14.5"
 val flink116V            = "1.16.0"
 val currentFlinkV        = "1.19.1"
 val sttpV                = "3.8.11"
 val kafkaV               = "3.3.1"
 val testContainersScalaV = "0.41.0"
 
-ThisBuild / version := "0.1-SNAPSHOT"
+ThisBuild / version := "1.0-nu1.18.0-SNAPSHOT"
 
 // todo: for now we should regularly bump the version until we start publish single "latest" -SNAPSHOT version
 val defaultNussknackerV = "1.18.0-staging-2024-09-24-20698-40fc17dbe-SNAPSHOT"
```

Comment:
Its default because it can be overriden by env variable from CI when testing triggered from main repo. I've moved this into nussknackerV scope to make it more clear.

Reply:
IMO Nu version should be hardcoded and every time it should be bumped manually (and the change should be reflected with a commit). Like we did in other projects that use Nu as a dependency.

Reply:
But then triggering tests from core repo would require pushing a commit to staging branch instead of triggering an action? Or do you have another solution in mind?